### DISCUSSION TITLE
2.x表单赋值不成功

### DISCUSSION THREAD

**Discussion by tangtang09** [alibaba/formily#1656] [Q&A 2.x] [answered]
![image](https://user-images.githubusercontent.com/83819365/123571312-8818f700-d7fc-11eb-83d0-032d0b03288e.png)
赋值使用form.setInitialValues( { supportEnsure: 'Aston Martin' ,maintainSupportTime:xxx} )失败，，加“requirement-block”这一层主要是为了给模块添加大标题，如下图：
![image](https://user-images.githubusercontent.com/83819365/123571635-3ae95500-d7fd-11eb-8133-dd197c6b9f97.png)

**请问是否有方法添加加大标题且不影响表单赋值？**

**Comment by janryWang** [comment]
给个可复现demo

**Reply by tangtang09** [reply]
https://codesandbox.io/s/bitter-wood-25svi?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
指定Card这个Field的type为void

**Reply by tangtang09** [reply]
请问这块文档说明有吗，

**Reply by janryWang** [reply]
需要看@formily/react和@formily/core文档，理解几个字段模型
